pkg/api/error.go

```diff
@@ -56,7 +62,7 @@ func (b *CloudErrorBody) String() string {
 		}
 	}
 
-	return fmt.Sprintf("%s: %s: %s%s", b.Code, b.Target, b.Message, details)
+	return fmt.Sprintf("%s: %s: %s: %s: %s%s", b.Code, b.Target, b.Message, b.Category, b.Dependency, details)
```

Comment:
for readability, lets put
code: %s, target: %s ...

Reply:
Sure thing, it would make it more clear for sure